### DISCUSSION TITLE
How to setup a monorepo with multiple prisma clients?

### DISCUSSION THREAD

**Discussion by abathellier** [prisma/prisma#18725] [Q&A] [answered]
We use [turboRepo](https://turbo.build/repo) for our monorepo app. And we have two apps with prisma at the same time. When we generate the two prisma clients, only one of them is generated because of hoisting.

How can we manage such a case? A solution would be to generate the prisma clients in a custom directory : https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client?query=monorepo&page=1

Is there a better way to do that? Especially if we want to move prisma in `packages` directory instead of `apps`.

**Comment by nurul3101** [comment, marked as answer]
Hello @durdenx 👋 

Is there any issue in generating PrismaClient in a custom directory?
I would have recommended you the same solution.

If it helps, here is an official example of [turborepo with prisma](https://github.com/vercel/turbo/tree/main/examples/with-prisma)

**Reply by abathellier** [reply]
No it’s fine. I just wanted to know if there was a better solution because I couldn’t find anything about it in the docs.

**Reply by nurul3101** [reply]
Ah, Okay.
In case I find any better solution, I'll post it here.
